.github/workflows/rebase-staging.yml

```diff
@@ -3,13 +3,13 @@ on:
     types:
       - created
 
-# This action allows people with write access to the repo to rebase a PRs base branch from
-# master to staging by commenting `/rebase-staging` on the PR while avoiding CODEOWNER notifications.
+# This action allows people with write access to the repo to rebase a PRs base branch
+# by commenting `/rebase ${branch}` on the PR while avoiding CODEOWNER notifications.
 
 jobs:
   rebase:
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'NixOS' && github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase-staging')
+    if: github.repository_owner == 'NixOS' && github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
```

Comment:
Should be even backwards compatible.

Reply:
It's not. We're expecting `/rebase <branch>` with this (see the `awk '{ print $2 }'`), while previously we accepted `/rebase-staging`.